src/converter/internal/compat/backendapi.cpp

```diff
@@ -50,11 +50,11 @@ using namespace mu::io;
 
 static const std::string PNG_WRITER_NAME = "png";
 static const std::string SVG_WRITER_NAME = "svg";
-static const std::string SEGMENTS_POSITIONS_WRITER_NAME = "sposXML";
-static const std::string MEASURES_POSITIONS_WRITER_NAME = "mposXML";
+static const std::string SEGMENTS_POSITIONS_WRITER_NAME = "spos";
+static const std::string MEASURES_POSITIONS_WRITER_NAME = "mpos";
```

Comment:
This change was not in #13122. Please remove it from this PR.

Reply:
done, will PR separately